### PR TITLE
Feature/hud redesign

### PR DESCRIPTION
**Pull request by samuelWilliams99**
image

### CODE REVIEW

**Review by plally — APPROVED** [review]
as long as  its been tested on a couple different resolutions :+1:

**Comment by samuelWilliams99** [comment]
> as long as its been tested on a couple different resolutions

Good point, will do tomorrow

**Comment by samuelWilliams99** [comment]
Tested on different resolutions, works fine

File: gamemode/modules/inventory/cl_inventory.lua [hunk 1]
```diff
@@ -295,6 +295,7 @@ function inv.setHotbarSlot( newIndex )
 end
 
 hook.Add( "InitPostEntity", "RVR_Inventory_HotbarSetup", inv.makeHotbar )
+hook.Add( "OnScreenSizeChanged", "RVR_Inventory_HotbarSetup", inv.makeHotbar )
```
Comment (overlordbleck) [inline gamemode/modules/inventory/cl_inventory.lua:298]: Can we check how many times this hook is called

**Review by overlordbleck — APPROVED** [review]
Looks good

File: gamemode/modules/inventory/cl_inventory.lua (hunk 1 above)
Comment (samuelWilliams99) [reply gamemode/modules/inventory/cl_inventory.lua:298]: once my guy

**Pull request merged**